docs/User/Frequently_asked_questions.md

```diff
@@ -315,8 +343,7 @@ external tools like pgAdmin).
 2. Then, run `<swarm-learning>/slm-ui/scripts/run-slm-ui -pw" supersecretpassword"`.
 
 ### What are the limitations of SLM-UI?
-Currently, SLM-UI works if you use Docker container runtime. SLM-UI does not work if you are running Podman container
-runtime on the host.<br>
+Currently, SLM-UI works if you use Docker container runtime.<br>
```

Comment:
Sorry please remove this above line as well..

"Currently, SLM-UI works if you use Docker container runtime.<br>"

Only have the last 2 lines..